.travis.yml

```diff
@@ -51,4 +51,4 @@ jobs:
     - stage: Code Quality
       env: STATIC_ANALYSIS
       install: travis_retry composer install --prefer-dist
-      script: vendor/bin/phpstan analyse -l 3 -c phpstan.neon lib tests
+      script: vendor/bin/phpstan a
```

Comment:
Yes. This is just how it is in other Doctrine libs so I made it consistent. And the other information like level and paths are in phpstan.neon.dist now to match other libs as well.

Reply:
to be exact, this is a shortcut for a command starting with `a`, as long as there is no ambiguity. That's the behavior of symfony/console. For a CI config, using the actual name is better IMO, as this avoids breaking the CI config in case a newer version adds another command starting with `a`

Reply:
I changed it to be `analyze`